.scripts/ApiCompat.ps1

```diff
@@ -0,0 +1,43 @@
+
+param(
+	[Parameter(Mandatory = $true)] [String]$BuildConfiguration
+)
+
+Write-Host "BuildConfiguration" $BuildConfiguration
+
+$apiTool="tool\.store\microsoft.dotnet.apicompat.tool\7.0.100\microsoft.dotnet.apicompat.tool\7.0.100\tools\net6.0\any\Microsoft.DotNet.ApiCompat.Tool.dll"        
+$apiToolRightFolder="Output"
+
+[xml]$xfiles = Get-Content .\ApiCompat.Files.xml
+$e = [char]0x274C
+
+foreach ($a in $xfiles.ApiCompat.Assembly)
+{
+    $n = $a.Name
+    $s = "suppress.$n.xml"
+    $l = $ExecutionContext.InvokeCommand.ExpandString($a.Location)
+    $apiToolLeft = "$l\$n.dll"
+    $apiToolRight = "$apiToolRightFolder\$n.dll"
+    
+    Set-Content $s "<Suppressions/>"
+
+    Write-Host "Checking" $n "..."
+    dotnet $apiTool --suppression-file $s --left-assembly $apiToolLeft --right-assembly $apiToolRight --enable-rule-attributes-must-match --enable-rule-cannot-change-parameter-name --generate-suppression-file
+    
+    [xml]$x = Get-Content $s
+    $e += [string]::Join([char]0x274C, ($x.Suppressions.Suppression | % { $_.DiagnosticId + " " + $_.Target })).Trim()
```

Comment:
Can we add a newline at the end of this?

Reply:
Tried various options and it didn't work as we use an env variable and they don't seem to support this type of character